po/common/fr.po

```diff
@@ -6360,17 +6360,17 @@ msgstr ""
 # Unit = element, not unit of measure
 msgctxt "packaging_number_of_units"
 msgid "Number of units"
-msgstr ""
+msgstr "Nombre d'éléments"
 
 # Unit = element, not unit of measure
 msgctxt "packaging_weight"
 msgid "Weight of one empty unit"
-msgstr ""
+msgstr "Poids d'un élément vide"
 
 # Unit = element, not unit of measure
 msgctxt "packaging_quantity_per_unit"
-msgid "Quantity of product contained per unit"
-msgstr ""
+msgid "Net quantity of product contained per unit"
+msgstr "Quantité nette de produit contenue par élément"
```

Comment:
```suggestion
msgstr "Quantité nette de produit contenue par élément (g)"
```

Reply:
See above, it can be ml, pieces, etc.

Reply:
Right, it's also why we use "Net quantity" instead of "Net weight": it can be a volume, a number of pieces (e.g. eggs) etc.

Also keep in mind this is a column name, it can't be too long, otherwise the display looks ugly.

Display with the changes from this PR:

![image](https://user-images.githubusercontent.com/8158668/210537711-97408b79-93a2-4a0b-abc7-f87505ac1c67.png)

Reply:
Ok fine, but you had to explain it to me. I find "net quantity" a bit fuzzy... I think we should use help bubbles as much as possible. I generally use the `title` attribute, which is good old HTML: https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/title
But I have just read that `title` isn't very suited for mobile, tablet, keyboard, etc. There is a good resource here:
https://inclusive-components.design/tooltips-toggletips/

Reply:
@CharlesNepote Good point, I filed https://github.com/openfoodfacts/openfoodfacts-server/issues/7939